WooCommerce/Classes/Yosemite/AuthenticatedState.swift

```diff
@@ -76,7 +76,8 @@ class AuthenticatedState: StoresManagerState {
             AnnouncementsStore(dispatcher: dispatcher,
                                storageManager: storageManager,
                                network: network,
-                               fileStorage: PListFileStorage())
+                               fileStorage: PListFileStorage()),
+            JetpackConnectionStore(dispatcher: dispatcher)
```

Comment:
❓ 
Is this being added here now because we were only handling .org site cred flow until now and that flow didn't replace `DeAuthenticatedState` instance with `AuthenticatedState` in `DefaultStoresManager`?

Reply:
Previously when I was working on the Jetpack connection handling for site credential login, the user is unauthenticated, so the store was only added to `DeauthenticatedState`. 

In this case, the user logged in to their WP.com account, so they are authenticated and the store has to be added to `AuthenticatedState` for its action to be handled properly.